.github/workflows/keycloak.yaml

```diff
@@ -0,0 +1,43 @@
+name: Keycloak AMD and ARM
+on:
```

Comment:
Should it be restricted to only run on main now?

Reply:
> Should it be restricted to only run on main now?

Not sure I am following, isn't it that this can only be triggered manually from Github Actions web interface? (or via `gh` command line)

Reply:
But you can run specify which branch to run it against using the `--ref` argument specified in the readme.
```gh workflow run build-multi-arch-image --ref <git_branch> -f keycloak_version=<desired_keycloak_version>```
I may have commented in the wrong place i.e. not suggesting adding a trigger but an excludes so that it can't be triggered against a branch.

For instance, I could create a branch and change the repository being build in the yaml and then trigger a workflow dispatch that would publish that image to our keycloak docker repo.

Reply:
I was about to say this is a moot point because if someone has access to push to a branch they could just remove the restriction as well. However my final thought would be that such a restriction wouldn't be a security measure it would be to show intent and stop someone unintentionally triggering it on a branch.

Reply:
If we decided to restrict it, it would need to be done as a condition that checks the ref. Something like  ```if: github.ref =='refs/heads/main'```

Reply:
Added the check. It seems to work. It still shows that the workflow execution were attempted (see screenshot below). This might be acceptable, as it's only supposed to prevent unintended triggering.

<img width="1423" alt="image" src="https://user-images.githubusercontent.com/91465696/214125628-2e79cb7d-9853-407a-9376-9ec5a3808151.png">